CoroutineTemplate/build.gradle.kts

```diff
@@ -0,0 +1,63 @@
+// Top-level build file where you can add configuration options common to all sub-projects/modules.
+
+buildscript {
+
```

Comment:
Redundant line break?

Reply:
@lydiasama It's from the old `build.gradle`. I will remove it.